erpnext/templates/includes/sticker_order_material_request.html

```diff
@@ -0,0 +1,51 @@
+<div class="tax-break-up" style="overflow-x: auto;">
+	<table class="table table-bordered table-hover">
+		<tbody>
+			{% for item in batch_doc %}
+			<tr>
+				<td>
+					{{ frappe.get_meta("Batch").get_label(item) }}
+				</td>
+				<td>
+					{{ batch_doc.get(item) or "--"}}
+				</td>
+			</tr>
+			{% endfor %}
+
+			{% for item in ci_doc %}
+			<tr>
+				<td>
+					{{ frappe.get_meta("Compliance Item").get_label(item) }}
+				</td>
+				<td>
+					{{ ci_doc.get(item) or "--"}}
+				</td>
+			</tr>
+			{% endfor %}
+
+			{% for item in reading_doc %}
+			<tr>
+				<td>
+					{{ item }}
+				</td>
+				<td>
+					{{ reading_doc.get(item) or "--" }}
+				</td>
+			</tr>
+			{% endfor %}
+
+			{% for item in qi_doc %}
+			<tr>
+				<td>
+					{{ frappe.get_meta("Quality Inspection").get_label(item) }}
+				</td>
+				<td>
+					{{ qi_doc.get(item) or "--" }}
+				</td>
+			</tr>
+			{% endfor %}
```

Comment:
Make this more compact:

```suggestion
			{% for item in batch_doc %}
				<tr>
					<td>{{ frappe.get_meta("Batch").get_label(item) }}</td>
					<td>{{ batch_doc.get(item) or "--" }}</td>
				</tr>
			{% endfor %}

			{% for item in ci_doc %}
				<tr>
					<td>{{ frappe.get_meta("Compliance Item").get_label(item) }}</td>
					<td>{{ ci_doc.get(item) or "--"}}</td>
				</tr>
			{% endfor %}

			{% for item in reading_doc %}
				<tr>
					<td>{{ item }}</td>
					<td>{{ reading_doc.get(item) or "--" }}</td>
				</tr>
			{% endfor %}

			{% for item in qi_doc %}
				<tr>
					<td>{{ frappe.get_meta("Quality Inspection").get_label(item) }}</td>
					<td>{{ qi_doc.get(item) or "--" }}</td>
				</tr>
			{% endfor %}
```